### ISSUE TITLE
Loss being same all the time

### ISSUE THREAD

**Issue by ozicay** [emo-box/EmoBox#12]
Hello again, thank you for your help. I solved the previous issue, but now I’m facing a new problem. When I start training, the loss values remain constant. However, when I comment out the condition in this line:

if self.check_gradients(): i comment it this line

self.optimizer.step()

the loss starts decreasing. I checked the check_gradients() method, and the gradients were never non-finite or problematic, so I decided to comment it out. İs that function necessary i just wonder
